frontend/src/components/ScheduledEventsTableRow/ScheduledEventsTableRow.tsx

```diff
@@ -12,7 +17,46 @@ export interface ScheduledEventsTableRowProps {
   start_date: string;
 }
 
+interface RunInstancePayload {
+  start_at: Moment;
+  status: string;
+}
+
+const BASEPATH = api.routes.VIEW_SCHEDULED_EVENTS;
+
+const createRunInstance = async (
+  scheduleId: number | undefined,
```

Comment:
This can be shorter ... and if you're getting complaints about not being able to have an undefined filed before an always defined one, change their positions.

Reply:
Now only accepts `number` removed `undefined` type